package.json

```diff
@@ -2,12 +2,12 @@
     "name": "flame-chart-js",
     "version": "2.0.1",
     "description": "",
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts",
+    "main": "dist/src/index.js",
+    "types": "dist/src/index.d.ts",
     "sideEffects": false,
     "scripts": {
         "build": "rollup -c",
-        "build:tsc": "tsc",
+        "build:tsc": "tsc -p ./tsconfig.build.json",
```

Comment:
You broke the build for npm publish.(

Reply:
How do i repro? Can you show me where to look?

Reply:
We have two builds - one for github pages and one for npm publishing. For the first one, we only provide a dist folder without any nesting, and for github pages, we only create an example folder. You build both.)
It would have been nice if the build had remained untouched, since I don't understand why these changes were made.)

Reply:
I believed there to be an issue - because when i do a npm build:tsc - the files generated do not map main or types in package.json.

I clearly don't understand your project as well as you - but my intentions were that i thought i was making a bug fix. I am sorry

![image](https://user-images.githubusercontent.com/1444266/211219690-27105a2a-4c75-4683-9251-079e1773826d.png)
![image](https://user-images.githubusercontent.com/1444266/211219738-c78a6f8b-c37a-4fb6-98dc-61e2738e091f.png)